src/ContactHub.js

```diff
@@ -87,6 +87,36 @@ const cleanCustomer = (data: APICustomer): Customer => {
   return customer;
 };
 
+/**
+ * Helper for adding 'page' to args.params object
+ */
+
+const buildArguments = (args: Object, page: number): Object => ({
```

Comment:
To make things clearer, I would modify this function to receive the `newPage` number instead of `+1` / `-1` (see other comments below...)